aten/src/ATen/native/native_functions.yaml

```diff
@@ -8616,33 +8616,14 @@
 - func: hardtanh.out(Tensor self, Scalar min_val=-1, Scalar max_val=1, *, Tensor(a!) out) -> Tensor(a!)
   device_check: NoCheck   # TensorIterator
   python_module: nn
-  dispatch:
-    CPU, CUDA: hardtanh_out
-    QuantizedCPU: hardtanh_out_quantized_cpu
```

Comment:
@bdhirsh do you think quantized would be a problem here? I see `clamp` has a quantized method, but not `clamp_out`, https://github.com/pytorch/pytorch/blob/9cdd1d7e489b17ba86aafdadef37eecbf0a64917/aten/src/ATen/native/native_functions.yaml#L1064

Reply:
Oh, you're right. So I think your two options are:

(a) Add a `QuantizedCPU` kernel for `clamp.out`. This should hopefully be pretty easy, since you can just use pretty much the same kernel currently used for `hardtanh.out`. Then you can fully remove the `hardtanh.out` entry

(b) Otherwise just leave the QuantizedCPU entry for `hardtanh.out`